debian/nfs-server-prep.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright 2019 Delphix
+#
+
+#
+# Called by the nfs-server.service unit before the server is started
+#
+
+#
+# DLPX-64900
+# Work around for kernel failing to parse the 'versions' input string
+# "-4.1 -4.2 -2 +3 +4" that is passed from rpc.nfsd after a reboot
+#
+# Prime the kernel version after a reboot to include +4 so that the -4.x works
+#
+# We need this work-around until Ubuntu picks up nfs-utils 2.3.3
+#
+versions=$(cat /proc/fs/nfsd/versions)
+if [[ "$versions" == "-2 -3 -4 -4.0 -4.1 -4.2" ]]; then
+	printf "+4\\n" >/proc/fs/nfsd/versions
+	echo "Initializing the NFS server version"
```

Comment:
Here and below, I think it would be nice to print the log message before we do the operation instead of after. If things hang or get interrupted, it makes it easier to know how far the service got.

Reply:
Good point.  I'll change the ordering.